src/nvim/ex_cmds.c

```diff
@@ -4960,3 +4961,20 @@ void ex_oldfiles(exarg_T *eap)
     }
   }
 }
+
+void ex_trust(exarg_T *eap)
+{
+  char *p = skiptowhite(eap->arg);
+  size_t len = (size_t)(p - eap->arg);
+  char *mode = xcalloc(len + 1, sizeof(char));
+  memcpy(mode, eap->arg, len);
+
+  char *path = skipwhite(p);
+  char *errmsg = nlua_trust(path, mode);
```

Comment:
I don't think there is any need for `nlua_trust` to be a separate function, it's probably not going to be used anywhere else. You can move `ex_trust` to `executor.c` and inline `nlua_trust` to make it all a single function

Reply:
Yeah, I was thinking about it, but wasn't sure if I wasn't supposed to have the `ex` function inside `ex_cmds`.
resolved in e3a4458632e45c6f897c39ee22c9236f044195b0

Reply:
I actually prefer the original approach. Functions should be grouped by interface, not implementation, and since `ex_trust` is an ex command it should be with the other ex commands. The fact that it's implemented using a Lua function is just an implementation detail, so I don't think it belongs in `executor.c`.

Reply:
Yes, that was my original thought process. On the other hand `executor.c` does contain `ex_lua` already.

Reply:
So what is the decision here? Personally I think I like having the lua wrapper in `executor.c` and all command related code in `ex_cmds.c` better.

Reply:
Split this into two separate functions again in 627dcefa7ed6e260fa64afe6616abbd06d225ac4